dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_contentlets_js_inc.jsp

```diff
@@ -174,7 +174,7 @@ final String calendarEventInode = null!=calendarEventSt ? calendarEventSt.inode(
 		**/
 		require([ "dijit/focus", "dojo/dom", "dojo/domReady!" ], function(focusUtil, dom){
 			dojo.require('dojox.timing');
-			t = new dojox.timing.Timer(500);
+			t = new dojox.timing.Timer(900);
```

Comment:
why did you increase it?

Reply:
It takes some time to render the search box.

Reply:
Maybe using `requestAnimationFrame` can work here. ([doc](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame))

Reply:
Noted

Reply:
I tried but it's not working.